translate/v31tov24/v31tov24.go

```diff
@@ -57,6 +57,15 @@ func Translate(cfg types.Config) (old.Config, error) {
 	// named by the path
 	fsList := generateFsList(cfg.Storage.Filesystems)
 
+	var httpProxy string
+	if cfg.Ignition.Proxy.HTTPProxy != nil {
+		httpProxy = *cfg.Ignition.Proxy.HTTPProxy
```

Comment:
I so miss the Rust `Option` + `unwrap_or_else()`...

Reply:
We have `util.StrV()`.

Reply:
OK thanks, definitely better.